packages/ketchup/src/managers/kup-dates/kup-dates-declarations.ts

```diff
@@ -40,3 +40,12 @@ export enum KupDatesFormats {
     ISO_TIME = 'HH:mm:ss',
     ISO_TIME_WITHOUT_SECONDS = 'HH:mm',
 }
+
+/**
+ * SortOrder
+ */
+
+export enum KupDateOrder {
```

Comment:
The syntax for managers' declarations should be `KupManagerclassDeclaration`.
In this case, `KupDatesOrder`.